journals/apps/journals/models.py

```diff
@@ -277,8 +277,8 @@ def __str__(self):
 # pylint: disable=wrong-import-position
 from .blocks import (
     JournalRichTextBlock, JournalImageChooserBlock, PDFBlock, XBlockVideoBlock,
-    PDF_BLOCK_TYPE, VIDEO_BLOCK_TYPE, IMAGE_BLOCK_TYPE, RICH_TEXT_BLOCK_TYPE,
-    STREAM_DATA_DOC_FIELD, STREAM_DATA_TYPE_FIELD)  # noqa
+    PDF_BLOCK_TYPE, VIDEO_BLOCK_TYPE, IMAGE_BLOCK_TYPE, RICH_TEXT_BLOCK_TYPE, RAW_HTML_BLOCK_TYPE,
+    STREAM_DATA_DOC_FIELD, STREAM_DATA_TYPE_FIELD, JournalRawHTMLBlock)  # noqa
```

Comment:
NIT: move JournalRawHTMLBlock in the section at the beginning of the import with the other non-constants